### PR TITLE
parser: Change syntax error log

### PR DESCRIPTION
**Pull request by shenli**
Make syntax log easy to read.

https://github.com/pingcap/tidb/issues/25

### CODE REVIEW

**Comment by ngaut** [comment]
LGTM

File: parser/scanner.l [hunk 1]
```diff
@@ -118,7 +118,7 @@ func (l *lexer) next() int {
 }
 
 func (l *lexer) err0(ln, c int, s string, args ...interface{}) {
-	err := errors.Errorf(fmt.Sprintf("%d:%d ", ln, c)+s, args...)
+	err := errors.Errorf(fmt.Sprintf("line %d column %d near \"%s\"", ln, c, l.val), args...)
```
Comment (qiuyesuifeng) [inline parser/scanner.l]: can variable s be removed from err0 function?

**Comment by Unknwon** [comment]
Much better, thanks!

**Comment by shenli** [comment]
PTAL @qiuyesuifeng

**Comment by qiuyesuifeng** [comment]
LGTM

**Pull request merged**